### DISCUSSION TITLE
How can I change the location of the blogsidebar to the left so that it has more space to display blog posts

### DISCUSSION THREAD

**Discussion by Shake-Jarjane** [facebook/docusaurus#8802] [Q&A] [answered]
I want to display sidebar when browsing blog articles, but the location of the sidebar is too close to the middle. I want to move it to the left side so that more space can be left to display blog articles

![image](https://user-images.githubusercontent.com/70839261/226893791-20b8f73c-06a7-49f6-a0eb-b97b37e75e0f.png)

**Comment by slorber** [comment, marked as answer]
You can swizzle all the React blog components and override their logic to implement the blog layout you want.

`BlogLayout` + `BlogSidebar/Desktop` in particular
